partner_delivery_link/models/res_partner.py

```diff
@@ -0,0 +1,34 @@
+from odoo import fields, models
```

Comment:
```suggestion
from odoo import fields, models, api
```

Reply:
@AungKoKoLin1997 I've applied changes can you please review it?